core/src/main/java/org/sonarsource/sonarlint/core/container/standalone/rule/StandaloneActiveRulesProvider.java

```diff
@@ -56,6 +69,22 @@ public ActiveRules provide(ProfileDefinition[] profileDefinitions) {
         registerProfilesForLanguage(builder, language, defs);
       }
 
+      for (Repository repo : ruleDefsLoader.getContext().repositories()) {
+        for (Rule rule : repo.rules()) {
+          if (rule.activatedByDefault()) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS134)